webapp/src/modules/publication/utils.js

```diff
@@ -0,0 +1,7 @@
+import { getContractAddress } from 'modules/wallet/utils'
+
+export function isLegacyPublication(publication) {
```

Comment:
Is it worth to put in share instead? Is there any context where we should use it server-side?

Reply:
Yes! but the problem was that I can't use getContractAddress, as it's using the `eth` singleton and that's not available on `shared/`
I've decided to leave it on publication/utils because it's "temporal" code anyways, it'll disappear with the old publications

Reply:
ok, cool! anways if we need it also server-side. We can refactor it sending the address as an input parameter